DEV-vtj-service.properties

```diff
@@ -0,0 +1,11 @@
+# nämä ovat 110% höpöä, mutta oltava määriteltyinä
+vtj.production.env=false
+vtj.keystore.file=/huu/haa/keystore.jks
+vtj.keystore.password=huuhaa
+vtj.truststore.file=/huu/haa/truststore.jks
+vtj.truststore.password=huuhaa
+vtj.cert.alias=huuhaa
+vtj.username=huuhaa
+vtj.password=huuhaa
+vtj.address=http://huu.haa
+cas.service.vtj-service=http://huu.haa
```

Comment:
seems that one required property is missing here, namely _host.cas_
after adding `host.cas=http://huu.haa` here everything worked _smooooothly_

Reply:
user error.